src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
+const resolved = document.createElement('div');
 
-// write your code here
+resolved.classList.add('message');
+resolved.textContent = 'Promise was resolved!';
+
+const rejected = document.createElement('div');
+
+rejected.classList.add('message');
+rejected.classList.add('message-error');
+rejected.textContent = 'Promise was rejected!';
+
+
+const logo = document.querySelector('.logo');
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', (e) => {
+    resolve('Success');
+    document.body.appendChild(resolved);
+  });
+});
```

Comment:
Promises should have explicit success and error handlers. Currently, the message is appended directly within the promise executor, which is not the correct approach. Consider using `.then()` and `.catch()` for handling promise resolution and rejection.